scripts/india_udise/common/base_school_data.py

```diff
@@ -104,6 +104,45 @@ class GeographicLevel(enum.Enum):
     BLOCK = "block"
 
 
+ATTRIBUTE_MAPPING = {
+    "cat1": {
+        "levelOfSchool": "PrimarySchool_Grade1To5"
```

Comment:
Would the prefix 'UDISE_' be added later?
The enums are: https://datacommons.org/browser/UDISE_PrimarySchool_Grade1To5

Reply:
Yes. In [this line](https://github.com/datacommonsorg/data/blob/307fd81d31edfecf1b4fc20564cc06f2014426de/scripts/india_udise/common/base_school_data.py#L290)